docs/0-what-is/1-getting-started.mdx

```diff
@@ -60,7 +60,7 @@ install any additional connector nor to change Conduit's default configuration.
 Next, we can use the Conduit CLI to build the example pipeline:
 
 ```shell
-$ conduit pipelines init
+$ conduit pipelines init <pipeline-name>
```

Comment:
make sure to update this command, with the example, after checking with the team